_docs/conversions.md

```diff
@@ -0,0 +1,113 @@
+---
+title: Conversion Processes
+permalink: /docs/publisher-tutorials/conversions/
+---
+
+## Write-API Conversion Process
```

Comment:
This section is a revelation to me. :) I can't find any references to it in the code.

1. if it's still applicable, then I think it needs more information -- what is the write-api-client? What is `client` in the next section.
2. If not applicable, I think we should remove this section all the way down to Converting File-based...

Reply:
This is something that we provided as part of our work on TRS. 
It hasn't been super popular, but I think its still a neat idea (i.e. an API that creates github repos and quay.io repos for tools and registers them into Dockstore given just the descriptors)

I would throw a better introduction to the idea here (can steal some text from https://wiki.oicr.on.ca/display/DOC/Dockstore+Write+API+%28Paste-bin+service%29+Specification ).

Reply:
https://github.com/dockstore/write_api_service/tree/develop  .  I'd be extremely surprised if it still works

Reply:
I'll try following along to see if I can get it working